### DISCUSSION TITLE
Just want to thank you for all the conributors.

### DISCUSSION THREAD

**Discussion by cerseinusantara** [AIrjen/OneButtonPrompt#44] [General]
Thank you. I love this extension.

**Comment by AIrjen** [comment]
Thank you! That is really appreciated. That is really kind of you.

I hope you have an awesome weekend.

**Reply by VirusShell** [reply]
Also wanted to say thank you as well! Love this extension and I absolutely love the ability to customize and fine-tune the lists that get used! I've spent a great deal of time adding to the lists, and in a few cases removing duplicated entries of tokens.

It also helps immensely that the code for this project is commented very well so I can see which pieces of what do this thing or that. Makes it easier if I want to tweak something or just try to understand what lead to a prompt getting created, as well as how it works in general.

Really enjoy this extension for the beautiful images that get created! I definitely appreciate the amount of work that's gone into this

**Reply by AIrjen** [reply]
Thank you for the compliment and support. This community is awesome and you guys are awesome for reaching out.

fun fact: I got lazy and the lists are deduplicated in the code 😅

If you have good additions for the lists, let me know. I’ll start a new topic here for that.
